src/duckling/canvas/drawing/drawn-construct.ts

```diff
@@ -1,13 +1,9 @@
-import {Container, DisplayObject, Point} from 'pixi.js';
+import {Container, DisplayObject, Point, Graphics} from 'pixi.js';
```

Comment:
As we discussed offline
I think DrawnConstruct should be a polymorphic interface with drawable?, paint?, and layer:string properties.

The loop used to draw entites should be replaced with
for (entity) for (attribute) constructs.push(getConstruct(attribute))
constructs.sort(construct.layer)

When drawing, you should create a new graphics object for each layer.